src/containers/Application/ApplicationPage.tsx

```diff
@@ -32,20 +32,36 @@ const ApplicationPage: React.FC<ApplicationProps> = ({ structure }) => {
 
   const currentSection = sectionCode
   const currentPage = `Page ${page}`
+  const currentSectionIndex = structure.sections[currentSection].details.index
+  const currentPageNumber = structure.sections[currentSection].pages[currentPage].number
 
   console.log('Structure', fullStructure)
 
   useEffect(() => {
-    if (!structure) return
+    if (!structure && !fullStructure) return
```

Comment:
I don't see why it's returning base on `fullStructure` not created but it only uses `structure` inside this effect... Should change to use `fullStructure`?

Reply:
66602ac